src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -138,6 +142,20 @@ def collect_data(geop_result, geojson):
     z.CN = np.array(geop_result['cn'])
     z.SedPhos = geop_result['sed_phos']
 
+    # Additional calculated values
+    basin_area_sq_km = area * SQKM_PER_SQM
+    z.SedDelivRatio = sediment_delivery_ratio(basin_area_sq_km)
```

Comment:
Since now the intention is conveyed by `SQKM_PER_SQM`, perhaps we don't need this second variable, and can just use
```python
z.SedDelivRatio = sediment_delivery_ratio(area * SQKM_PER_SQM)
```